nipap/nipap/backend.py

```diff
@@ -758,17 +763,53 @@ def _connect_db(self):
                 del(db_args[key])
 
         # Create database connection
-        try:
-            self._con_pg = psycopg2.connect(**db_args)
-            self._con_pg.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
-            self._curs_pg = self._con_pg.cursor(cursor_factory=psycopg2.extras.DictCursor)
-            self._register_inet()
-            psycopg2.extras.register_hstore(self._con_pg, globally=True, unicode=True)
-        except psycopg2.Error as exc:
-            self._logger.error("pgsql: %s" % exc)
-            raise NipapError("Backend unable to connect to database")
-        except psycopg2.Warning as warn:
-            self._logger.warning('pgsql: %s' % warn)
+        while True:
```

Comment:
```while True:``` without any kind of throttling always rings some warning bells for me, but I guess we're good here :smile: